bin/devdata.py

```diff
@@ -0,0 +1,25 @@
+"""Copy tox environment variables from devdata."""
```

Comment:
```suggestion
"""Download .devdata.env from github.com:hypothesis/devdata.git."""
```

Although our other apps use a tox plugin to read `.devdata.env` into tox envs, `.devdata.env` itself is just a standard env file, doesn't have anything in particular to do with tox